### DISCUSSION TITLE
PocketHost 0.9.2: Enhanced Security & Streamlined Experience

### DISCUSSION THREAD

**Discussion by benallfree** [pockethost/pockethost#314] [Show and tell]
_Unveil the latest patches, keen additions, and crucial fixes in v0.9.2 for PocketHost.io, honing its seamless performance. Get the scoop right from reversing linting errors to Cloudflare proxy enforcement for rock-solid security. It's all set for a more robust, secure, and agile developer experience. Let's deep-dive!_

Hot off the digital press, welcome to the upgraded version of PocketHost, v0.9.2! Like an episode of 'Silicon Valley' meets 'Mr. Robot,' this upgraded experience is reliable, intuitive, and smoother than butter on a hot knife. We've been beavering away behind the scenes, embracing the profound power of `rizzdown` to AI-enhance our docs and blogs. Just as Neo sees the Matrix, our content is now rizz'd, bringing you clarity amidst complexity, conveniently affixed with community discussion links.

We've fixed and finessed to enhance your user experience. With bug-bashing efforts on linting errors using prettier, we've added JSON file types to linting, smoothed over registration processes and GitHub Actions, and even added `patch-package` post-processing to all monorepo packages. Imagine the power of Mjolnir in web development, literally raining down patches in all the right places.

Our latest dashboard is a work of art, fully supporting a full-screen logging view for instances. It wears a new suit, tailored in the DaisyUI and Tailwind framework, adding a razzle-dazzle element to your working experience. We turned the protective dial up to 'John Wick' levels by introducing IPCIDR filtering – now all traffic must go through the Cloudflare proxy for comprehensive DDos and security coverage.

The biggest game-changer is the simplified proxy layers from the PocketHost daemon, enhancing reliability to match the highest level of coder zen that comes from less code. With OpenGraph metadata enhancements and a `return to top` button, we've summoned Avenger-like levels of user-experience power. Try out the new features and immerse yourself in a strengthened, simplified, and slicker PocketHost world.

# Change Log

- Version 0.9.2 (patch)
- Fixed Github Actions to properly publish the www static site when commits to the master branch are made
- Started using `rizzdown` to generate AI-enhanced docs and blog posts. All content has been updated with rizz'd versions
- Added community discussion links to the bottom of all blog posts
- Fixed linting errors using prettier
- Added JSON file types to linting
- Added `patch-package` postprocessing to all monorepo packages
- fix: github actions memo output formatting on master
- enh: github actions to only notify Discord if files have actually changed
- Added LICENSE.md and small edits in readme.md (#312)
- Removed all references to the old gitbook documentation
- Dashboard - Sidebar Maintenance &amp; Confirm Account Banner (#310)
- Dashboard now supports full screen logging view for instances
- Added a `return to top` button on all blog and documentation pages
- Fixed bug preventing registration and signup
- Added OpenGraph metadata enhancements to all blog and documentation pages
- Deployed a new dashboard using the DaisyUI and Tailwind frameworks
- Added `prettier` hooks on build steps in `husky` hooks
- Added IPCIDR filtering. Now all traffic must go through the Cloudflare proxy. The origin will reject it otherwise. This is important for DDoS and other security measures.
- Removed the top-level proxy from the PocketHost daemon, thus simplifying the number of layers involved in proxying traffic to instances. Fewer layers and less code means more reliability.
- Updated the PocketBase JS SDK to the latest version
- Encapsulated HTTP/HTTPS into a variable for an easier developer experience
- Fixed CORS headers in realtime logging code
- Fixed documentation links

**Comment by Graciasjr** [comment]
> _Unveil the latest patches, keen additions, and crucial fixes in v0.9.2 for PocketHost.io, honing its seamless performance. Get the scoop right from reversing linting errors to Cloudflare proxy enforcement for rock-solid security. It's all set for a more robust, secure, and agile developer experience. Let's deep-dive!_
> 
> Hot off the digital press, welcome to the upgraded version of PocketHost, v0.9.2! Like an episode of 'Silicon Valley' meets 'Mr. Robot,' this upgraded experience is reliable, intuitive, and smoother than butter on a hot knife. We've been beavering away behind the scenes, embracing the profound power of `rizzdown` to AI-enhance our docs and blogs. Just as Neo sees the Matrix, our content is now rizz'd, bringing you clarity amidst complexity, conveniently affixed with community discussion links.
> 
> We've fixed and finessed to enhance your user experience. With bug-bashing efforts on linting errors using prettier, we've added JSON file types to linting, smoothed over registration processes and GitHub Actions, and even added `patch-package` post-processing to all monorepo packages. Imagine the power of Mjolnir in web development, literally raining down patches in all the right places.
> 
> Our latest dashboard is a work of art, fully supporting a full-screen logging view for instances. It wears a new suit, tailored in the DaisyUI and Tailwind framework, adding a razzle-dazzle element to your working experience. We turned the protective dial up to 'John Wick' levels by introducing IPCIDR filtering – now all traffic must go through the Cloudflare proxy for comprehensive DDos and security coverage.
> 
> The biggest game-changer is the simplified proxy layers from the PocketHost daemon, enhancing reliability to match the highest level of coder zen that comes from less code. With OpenGraph metadata enhancements and a `return to top` button, we've summoned Avenger-like levels of user-experience power. Try out the new features and immerse yourself in a strengthened, simplified, and slicker PocketHost world.
> 
> # Change Log
> * Version 0.9.2 (patch)
> * Fixed Github Actions to properly publish the www static site when commits to the master branch are made
> * Started using `rizzdown` to generate AI-enhanced docs and blog posts. All content has been updated with rizz'd versions
> * Added community discussion links to the bottom of all blog posts
> * Fixed linting errors using prettier
> * Added JSON file types to linting
> * Added `patch-package` postprocessing to all monorepo packages
> * fix: github actions memo output formatting on master
> * enh: github actions to only notify Discord if files have actually changed
> * Added LICENSE.md and small edits in readme.md ([Added LICENSE.md and small edits in readme.md #312](https://github.com/pockethost/pockethost/pull/312))
> * Removed all references to the old gitbook documentation
> * Dashboard - Sidebar Maintenance & Confirm Account Banner ([Dashboard - Sidebar Maintenance & Confirm Account Banner #310](https://github.com/pockethost/pockethost/pull/310))
> * Dashboard now supports full screen logging view for instances
> * Added a `return to top` button on all blog and documentation pages
> * Fixed bug preventing registration and signup
> * Added OpenGraph metadata enhancements to all blog and documentation pages
> * Deployed a new dashboard using the DaisyUI and Tailwind frameworks
> * Added `prettier` hooks on build steps in `husky` hooks
> * Added IPCIDR filtering. Now all traffic must go through the Cloudflare proxy. The origin will reject it otherwise. This is important for DDoS and other security measures.
> * Removed the top-level proxy from the PocketHost daemon, thus simplifying the number of layers involved in proxying traffic to instances. Fewer layers and less code means more reliability.
> * Updated the PocketBase JS SDK to the latest version
> * Encapsulated HTTP/HTTPS into a variable for an easier developer experience
> * Fixed CORS headers in realtime logging code
> * Fixed documentation links

Hello @benallfree  First of all, I would like to tell you how much I love and am excited about the pockhost and Pocketbase project. Thank you for putting this project together. So recently, I have been using Pocketbase as a backend in my projects and pockhost for hosting because of the freeness and ease of use. Currently I am developing a project which I intend to publish on a large scale and I hope to collect a lot of users. But I have a concern, is pockethost a project that will last over time and can I use it in production.. Ps, I would love to use it in production and even if in the future I will have to pay packages to use it. Additionally, I love the idea of contributing to building an ecosystem around Pocketbase. Thank you for answering me 🙏

**Reply by benallfree** [reply]
Hi @Graciasjr, PocketHost is free to use until version 1.0 when a paid tier will be introduced. However, there is no timeline for that. People are using PocketHost now for production applications and you can always easily migrate away from the platform and host things yourself if you decide you want to later. I hope those answers help!

**Reply by Graciasjr** [reply]
yes thank so much, it helped me 🙌🏾
